### PR TITLE
root: 6.32.00 -> 6.32.02

### PR DESCRIPTION
**Pull request by guitargeek**
* update ROOT to latest patch release 6.32.02
* don't explicitly set `tmva-pymva=OFF`: building this component of ROOT doesn't require anything that is not
already in the dependencies

### CODE REVIEW

File: pkgs/applications/science/misc/root/default.nix [hunk 1]
```diff
@@ -56,15 +56,15 @@
...
     tests = import ./tests { inherit callPackage; };
   };
 
   src = fetchurl {
     url = "https://root.cern.ch/download/root_v${version}.source.tar.gz";
-    hash = "sha256-EvIDaBpZBBxHTOlSN2Hm8OiGGzvueN9feZqNtVGJ5dI=";
+    hash = "sha256-PQ92vwWFfhgHzPssngFPUlvLYl+UojcLRV9LFklhYC0=";
   };
 
   clad_src = fetchgit {
```
Comment (Artturin) [inline pkgs/applications/science/misc/root/default.nix:70]: Add a link to https://github.com/root-project/root/blob/master/interpreter/cling/tools/plugins/clad/CMakeLists.txt#L76 above the ref

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 322283` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>hepmc3</li>
    <li>ns-3</li>
    <li>python311Packages.hepmc3</li>
    <li>python312Packages.hepmc3</li>
    <li>root</li>
    <li>xfitter</li>
    <li>yoda-with-root</li>
  </ul>
</details>

**Comment by Artturin** [comment]
It appears to be building clang as a subproject, can it be unvendored?

**Comment by veprbl** [comment]
> It appears to be building clang as a subproject, can it be unvendored?

No, that's not possible.

**Comment by veprbl** [comment]
```
/tmp/nix-build-root-6.32.02.drv-0/root-6.32.02/core/newdelete/src/NewDelete.cxx:303:13: error: aligned allocation function of type 'void *(std::size_t, std::align_val_t)' is only available on macOS 10.13 or newer
   return ::operator new(size, (std::align_val_t)__STDCPP_DEFAULT_NEW_ALIGNMENT__);
            ^
/tmp/nix-build-root-6.32.02.drv-0/root-6.32.02/core/newdelete/src/NewDelete.cxx:303:13: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
1 error generated.
```
is what fails on macOS

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 322283` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>hepmc3</li>
    <li>ns-3</li>
    <li>python311Packages.hepmc3</li>
    <li>python312Packages.hepmc3</li>
    <li>root</li>
    <li>xfitter</li>
    <li>yoda-with-root</li>
  </ul>
</details>

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 322283` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>hepmc3</li>
    <li>ns-3</li>
    <li>python311Packages.hepmc3</li>
    <li>python312Packages.hepmc3</li>
    <li>root</li>
    <li>root.debug</li>
    <li>xfitter</li>
    <li>yoda-with-root</li>
  </ul>
</details>

**Review by veprbl — APPROVED** [review]

**Comment by guitargeek** [comment]
Hi @veprbl, thanks for the updates! The PR is ready to be merged now as far as I can tell. However, I can't seem to be able to merge this PR myself. Can you make do it? Thanks!

**Pull request merged**